<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersList</class>
 <widget class="QMainWindow" name="UsersList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список пользователей</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="btnBan">
       <property name="text">
        <string>В бан</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnEnableDisable">
       <property name="text">
        <string>Отключить</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QListView" name="listUsers"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
